sql/core/src/test/scala/org/apache/spark/sql/sources/TableScanSuite.scala

```diff
@@ -314,4 +314,20 @@ class TableScanSuite extends DataSourceTest {
       sql("SELECT * FROM oneToTenDef"),
       (1 to 10).map(Row(_)).toSeq)
   }
+
+  test("schema field with comment") {
+    sql(
+      """
+       |CREATE TEMPORARY TABLE student(name string comment "the name of a student")
```

Comment:
Can we update the schema to have 2 or more columns (just want to make sure the comma separator works well) and to have columns with and without comments?